model/appender_test.go

```diff
@@ -1819,3 +1823,149 @@ func TestSignatureAppearanceWithTimestamp(t *testing.T) {
 	}
 	validateFile(t, tempFile("appender-signature-appearance-with-timestamp.pdf"))
 }
+
+func TestValidatePAdESSignature(t *testing.T) {
+
+	validateFile(t, "./testdata/dss/pades-5-signatures-and-1-document-timestamp.pdf")
+	validateFile(t, "./testdata/dss/Test.signed_Certipost-2048-SHA512.extended-LTA.pdf")
+
+	validateFile(t, "./testdata/dss/doc-firmado.pdf")
+	validateFile(t, "./testdata/dss/doc-firmado-LT.pdf")
+	validateFile(t, "./testdata/dss/doc-firmado-T.pdf")
+	validateFile(t, "./testdata/dss/DSS-1443.pdf")
+	validateFile(t, "./testdata/dss/DSS-1683.pdf")
+
+	validateFile(t, "./testdata/dss/hello_signed_INCSAVE_signed.pdf")
+
+	validateFile(t, "./testdata/dss/modified_after_signature.pdf")
+
+	validateFile(t, "./testdata/dss/PAdES-LTA.pdf")
+	validateFile(t, "./testdata/dss/pdf-signed-original.pdf")
+	validateFile(t, "./testdata/dss/pkcs7.pdf")
+
+	validateFile(t, "./testdata/dss/siwa.pdf")
+	validateFile(t, "./testdata/dss/Test.signed_Certipost-2048-SHA512.extended-LTA.pdf")
+	validateFile(t, "./testdata/dss/TEST2_signed_with_zero_policy_hash.pdf")
+	validateFile(t, "./testdata/dss/TestToSignPDFSHA256_TST_SIG_NOT_FOUND.pdf")
+
+}
+
+func TestAppenderSignPAdES(t *testing.T) {
+	pass := os.Getenv("TINYCERT_USER_PASS")
```

Comment:
is it possible to automate this test better?
or we can define some static/test credentials?

Reply:
I changed the certificate password to default.